spec/features/two_factor_authentication/multiple_mfa_sign_up_spec.rb

```diff
@@ -250,30 +234,31 @@
 
       expect(page).to have_current_path backup_code_setup_path
 
-      click_continue
-
       expect(page).to have_link(t('components.download_button.label'))
+    end
 
+    it 'shows the confirm backup codes page' do
       click_continue
 
       expect(page).to have_current_path(
-        auth_method_confirmation_path,
+        confirm_backup_codes_path,
       )
-
-      click_button t('mfa.skip')
     end
 
     it 'goes to the next page after user confirms that they have saved their backup codes' do
-      acknowledge_backup_code_confirmation
+      click_continue
+      expect(page).to have_current_path(
+        confirm_backup_codes_path,
+      )
+
+      click_continue
       expect(page).to have_current_path account_path
     end
 
-    it 'regenerates backup codes path if a user clicks that they need new backup codes' do
-      find(
-        'a',
-        text: t('two_factor_authentication.backup_codes.new_backup_codes_html').gsub('&nbsp;', ' '),
-      ).click
-      expect(page).to have_current_path backup_code_regenerate_path
+    it 'returns to setup mfa page when user clicks Cancel' do
+      click_on(t('links.cancel'))
+      expect(current_path).to eq account_two_factor_authentication_path
+      expect(mfa.backup_code_configurations).to be_empty
```

Comment:
I have coverage for cancel -> delete, and so on in this test as part of a preexisting backup codes block.

Reply:
It should be working now. I was missing something in backup_code_setup_controller at authorize_backup_code_disable